Java/androidTest/java/com/couchbase/litecore/C4DatabaseTest.java

```diff
@@ -396,7 +444,7 @@ public void testDatabaseCancelExpire() throws LiteCoreException {
         // unix time
         long expire = System.currentTimeMillis() / 1000 + 2;
         db.setExpiration(docID, expire);
-        db.setExpiration(docID, Long.MAX_VALUE);
+        db.setExpiration(docID, null);
```

Comment:
After sleeping for 2 seconds, try to purge expired doc and get the doc to see if the doc is still there?

Reply:
Also please remove TODO below as well.

Reply:
Should it be db.setExpiration(docID, 0); instead of null?